src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -464,15 +465,36 @@ public Task GetMigratedServerUrls()
             return Task.CompletedTask;
         }
 
+        [RavenAction("/admin/smuggler/migrator-path", "GET", AuthorizationStatus.Operator)]
+        public Task GetConfigurationMigratorPath()
+        {
+            // If the path from the configuration is defined, the Studio will block the option to set the path in the import view
+            using (Database.DocumentsStorage.ContextPool.AllocateOperationContext(out DocumentsOperationContext context))
+            {
+                using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+                {
+                    writer.WriteStartObject();
+                    writer.WritePropertyName("PathDefined");
```

Comment:
nameof(MigrationConfiguration.MigratorPath)

Reply:
I think we don't want to expose the actual path, only if it is defined or not.
So I'm sending a boolean here.